modules/dev-tools/package.json

```diff
@@ -41,12 +41,16 @@
     "@babel/core": "^7.8.7",
     "@babel/plugin-transform-runtime": "^7.8.7",
     "@babel/preset-env": "^7.8.7",
+    "@typescript-eslint/eslint-plugin": "^4.14.1",
```

Comment:
We should bump the package.json version to `0.3.0`.

Reply:
Done. Shouldn't we add `-alpha.0`?
BTW, can we test it without publishing.